tests/integration/test_remote_indexes.py

```diff
@@ -1,7 +1,7 @@
 # This file contains tests that actually query remote package indexes,
 # to ensure that micropip works with real-world package indexes.
 # Since running these tests will send many requests to remote servers,
-# these tests are disabled by default.
+# these tests are disabled by default in CI.
```

Comment:
Above you said: "To run these tests locally, add `--integration` flag". Here you say they are disabled by default _in CI_. But aren't they are also disabled by default locally?

Reply:
Right, they are disabled both locally and CI by default. I'll update the wording